openeo_processes_dask/process_implementations/cubes/load.py

```diff
@@ -0,0 +1,164 @@
+import datetime
+import json
+from collections.abc import Iterator
+from pathlib import PurePosixPath
+from typing import Any, Callable, Dict, List, Optional, Tuple, Union
+from urllib.parse import unquote, urljoin, urlparse
+
+import planetary_computer as pc
+import pyproj
+import pystac_client
+import stackstac
+import xarray as xr
+from openeo_pg_parser_networkx.pg_schema import *
```

Comment:
I think star imports like this should be avoided, especially from a dependency package. I have no idea what this line imports